network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -663,7 +682,16 @@ def total_vote_count(self):
 
     @property
     def creepiness(self):
-        return self.evaluation.average_creepiness
+        try:
+            # Try an annotation made above the ProductPage level
+            return self._average_creepiness
```

Comment:
**Suggestion:** This logic might be simpler if the `Prefetch` annotation was used as above - you'd only need to check for the attribute on the Evaluation then.